cmd/cat/cat.go

```diff
@@ -0,0 +1,49 @@
+// Based on Plan9 cat
+
+package main
+
+import (
+	"log"
+	"os"
+)
+
+func cat(f *os.File, name string) {
+	var (
+		buf [8192]byte
+	)
+
+	for {
+		n, err := f.Read(buf[:])
+
+		if err != nil {
+			log.Fatal("error reading %s: %s", name, err)
```

Comment:
You are right! It should be Fatalf. =)

But using the "log" package for this is not good anyway. I'll update the code to use a simple print+exit